HotelPriceScout/Data/Interface/Dashboard.cs

```diff
@@ -24,16 +26,63 @@ public class Dashboard
         public DateTime LastDayOfMonth { get; set; } = new DateTime(DateTime.Now.Year, DateTime.Now.AddMonths(1).Month, 1).AddDays(-1);
         private SqliteDataAccess _db = new SqliteDataAccess();
 
+        public async Task<IEnumerable<MarketPriceModel>> DisplaySelectedComparedPrices(List<string> SelectedHotels, string StartDate, string EndDate, int RoomType)
+        {
+            if (SelectedHotels.Contains("Local"))
+            {
+                SelectedHotels.Add("Cabinn Aalborg");
+                SelectedHotels.Add("Slotshotellet Aalborg");
+                SelectedHotels.Add("Kompas Hotel Aalborg");
+            }
+            if (SelectedHotels.Contains("No budget"))
+            {
+                SelectedHotels.Add("Kompas Hotel Aalborg");
+                SelectedHotels.Add("Slotshotellet Aalborg");
+                SelectedHotels.Add("Milling Hotel Aalborg");
+                SelectedHotels.Add("Aalborg Airport Hotel");
+                SelectedHotels.Add("Helnan Phønix Hotel");
+                SelectedHotels.Add("Hotel Schellsminde");
+                SelectedHotels.Add("Radisson Blu Limfjord Hotel Aalborg");
+                SelectedHotels.Add("Comwell Hvide Hus Aalborg");
+                SelectedHotels.Add("Scandic Aalborg Øst");
+                SelectedHotels.Add("Scandic Aalborg City");
+            }
+            var last = SelectedHotels.LastOrDefault();
+            foreach (var item in SelectedHotels)
+            {
+                if (item.Equals(last))
+                {
+                    All += "'";
+                    All += item;
+                    All += "'";
+                }
+                else
+                {
+                    All += "'";
+                    All += item;
+                    All += "'";
+                    All += " OR ";
+                    All += "HotelName = ";
```

Comment:
Du kunne bare altid tilføje de 3 første ting til All. Så kan du bagefter have et if statment der tilføjer de sidste to til All